De novo repeat annotation documnet.tex

```diff
@@ -1,321 +1,508 @@
+	.
+	\begin{lstlisting}[language=bash]
+	phobos-linux-gcc4.1.2 -r 7 --outputFormat 0 --printRepeatSeqMode 0 notKnownNotProtein.fa > notKnownNotProtein.phobos
+	\end{lstlisting}
+	
+	\subsubsection{Identify the sequences that are SSRs from the phobos output.}
+	phobos output will be used to identify SSRs: notKnownNotProtein.phobos.
+	\begin{lstlisting}[language=bash]
+	javac IdentifySSRs.java
+	java IdentifySSRs
+	\end{lstlisting}
+	Your output will be a file called: SSR.txt
+	
+	\subsubsection{Generate annotated repeat library.}
+	There are ten input files that are required to generate a repeat library: \\
+	1. ConsensusSequences.fa \\
+	2. ConsensusSequences.fa.map \\
+	3. notKnown.fa.tewb.gff \\
+	4. notKnown.fa.ervwb.gff \\
+	5. protein.txt \\
+	6. known.txt \\
+	7. GB\_TE.21032016.fa \\
+	8. all\_retrovirus.fasta \\
+	9. SSR.txt (if you do not have this, leave the definition in, it will generate error messages, but will not stop the program or affect the results.) \\
+	10. LA4v2-satellite.fa (you do not have this, or equivalent, as you didn't have any satellites, but leave the definition in-it will cause error messages, but will not stop the program or affect the results.)
+	\begin{lstlisting}[language=java]
+	javac GenerateAnnotatedLibrary.java
+	java GenerateAnnotatedLibrary
+	\end{lstlisting}
+	This will generate a library called \textquotedblleft Human\_Repeat\_Library.fasta\textquotedblright, you can change this to whatever you want to call your library.
+	
+	\subsection{Benchmarks2}
+	
+	\footnotesize  % Switch from 12pt to 11pt; otherwise, table won't fit
+	%\setlength\LTleft{-50pt}            % default: \fill
+	%\setlength\LTright{-50pt}           % default: \fill
+	\setlength\tabcolsep{1.5pt}
+	\begin{center}
+		\begin{tabular}{ | c | c | c | c| c |c|}
+			\hline
+			\thead{Genome}	& 	\thead{Consensus sequences size}	&	\thead{Censor first run \\ time (hh:mm)}	&	\thead{reportJ.sh \\ (hh:mm)}	&	\thead{phobos run time \\ (hh:mm)} \\
+			\hline
+			Human	&	38M	&	5:14	&	19:30	&	$<$00:10 \\
+			\hline 
+			Bearded Dragon	&	88M	&	22:21	&	$<$178	&	$<$00:10 \\
+			\hline
+			\textcolor{red}{New Bearded Dragon}	&	\textcolor{red}{88M}	&	\textcolor{red}{7:13}	&	\textcolor{red}{18:28}	&	\textcolor{red}{$<$00:10} \\
+			\hline
+			Anolis	&	63M	&	9:42	&	78	&	$<$00:10 \\
+			\hline
+			Chicken	&	18M	&	3:01	&	$<$24	&	$<$00:10 \\
+			\hline
+			Opossum	&	60M	&	13:17	&	80	&	$<$01:00 \\
+			\hline
+			Platypus	&	162M	&	17:34	&	115	&	$<$01:00 \\
+			\hline
+			Echidna	&	59M	&	18:40	&	105	&	01:54 \\
+			\hline
+		\end{tabular}
+	\end{center}
+	
+	\noindent * Analysis run on a slurm machine with 4$\sim$16 cpus, and 8GB RAM, running Red Hat Linux.\\
+	\textcolor{red}{** New Bearded dragon analysis used same bearded dragon genome, except it was run on a High Performance Computing machine with 32 cpus, running Red Hat Linux.} \\
```

Comment:
Is there a particular reason this needs to be red?

Reply:
Done